File: play-2.4.x/K-Modules/kmodules/K-Base/app/models/K/EbeanConfig/PersistLog.java
package models.K.EbeanConfig;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.databind.JsonNode;

import java.util.Date;

/**
 * Created by kk on 14/11/28.
 */
public class PersistLog {

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss.SSS", timezone = "Asia/Shanghai")
    public Date action_time;
    public String action_type;
    public String bean_class;
    public JsonNode bean_data;

    public PersistLog() {
        action_time = new Date();
    }
}

File: play-kotlin-2.5.x/PlayKotlin/kmodules/K-Base/app/models/K/EbeanConfig/PersistLog.kt
package models.K.EbeanConfig

import com.fasterxml.jackson.annotation.JsonFormat
import com.fasterxml.jackson.databind.JsonNode

import java.util.Date

/**
 * Created by kk on 14/11/28.
 */
class PersistLog {

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss.SSS", timezone = "Asia/Shanghai")
    var action_time: Date
    var action_type: String? = null
    var bean_class: String? = null
    var bean_data: JsonNode? = null

    init {
        action_time = Date()
    }
}
